src/coreclr/jit/compiler.h

```diff
@@ -9606,7 +9681,8 @@ class Compiler
     //
     bool canUseEvexEncoding() const
     {
-        return compOpportunisticallyDependsOn(InstructionSet_AVX512F);
+        return (compOpportunisticallyDependsOn(InstructionSet_AVX10v1) ||
```

Comment:
Same here, this can just check `InstructionSet_EVEX` now

Reply:
Changed.